src/main/java/co/inventorsoft/StreamHomework.java

```diff
@@ -49,7 +65,10 @@
      * @return map {age : people with this age}
      */
     public Map<Integer, List<Person>> groupByAge(final List<Person> people) {
-        return null;
+        System.out.println(people);
```

Comment:
Such "logging" shouldn't end up in the final code 👀 
(no need to change it now, just next time make sure you cleaned up the code before committing:))